src-python/lock-keeper/lockkeeper.py

```diff
@@ -80,37 +86,37 @@ def parse_dump_flows(raw):
 
 @app.route('/flows', methods=['GET'])
 def get_flows_route():
-    flows = execute_remote_command('ovs-ofctl dump-flows br0')
-    return jsonify(parse_dump_flows(flows))
+    switch_cli = _get_switch_cli(request.args.get(SWITCH_TYPE_PARAMETER))
+    return jsonify(switch_cli.dump_all_flows())
 
 
 @app.route('/flows', methods=['POST'])
 def post_flows_route():
     payload = request.get_json()
-    commands = ['ovs-ofctl add-flow br0 in_port={in_port},' \
-                'actions=output={out_port}'.format(**flow) for flow in payload]
-    execute_remote_commands(commands)
+    switch_cli = _get_switch_cli(request.args.get(SWITCH_TYPE_PARAMETER))
+    switch_cli.create_flows(payload)
```

Comment:
If you pass `request` instead of particular argument into `_get_switch_cli()` and avoid temporary variables, you'll be able to shorten the code to 
```
_get_switch_cli(request).create_flows(payload)
```